gradle/dependency-locks/compileClasspath.lockfile

```diff
@@ -39,8 +40,6 @@ org.glassfish:javax.json:1.1
 org.jetbrains.kotlin:kotlin-stdlib-common:1.2.70
 org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.70
 org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.70
-org.jetbrains.kotlin:kotlin-stdlib-jre7:1.2.70
-org.jetbrains.kotlin:kotlin-stdlib-jre8:1.2.70
```

Comment:
The JPT lib bumps included the fixes for https://ecosystem.atlassian.net/browse/JPERF-466